.github/actions/setup-go/action.yml

```diff
@@ -15,6 +15,15 @@ runs:
         cache-dependency-path: cli/go.sum
 
     - name: Set Up Protoc
+      id: set-up-protoc
+      continue-on-error: true
+      uses: arduino/setup-protoc@v1
+      with:
+        version: "3.x"
+        repo-token: ${{ inputs.github-token }}
+
+    - name: Set Up Protoc (second try)
+      if: steps.set-up-protoc.outcome == 'failure'
```

Comment:
I'm a bit sad about the body of these actions being repeated twice, but GH actions not having an integrated retry mechanism is really to blame here.